charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -79,18 +79,15 @@ spec:
           {{- range .Values.backstage.args }}
             - {{ . | quote }}
           {{- end }}
-          {{- if .Values.backstage.resources }}
-          resources: {{- include "common.tplvalues.render" (dict "value" .Values.backstage.resources "context" $) | nindent 12 }}
-          {{- end }}
           {{- if .Values.backstage.extraAppConfig }}
           {{- range .Values.backstage.extraAppConfig }}
             - "--config"
-            - {{ .filename | quote }}
+            - "{{ .Values.backstage.installDir }}/{{ .filename }}"
```

Comment:
`.Values.backstage.installDir` is not within the scope of the `range` block. Installing the chart will fail with an error saying  _<.Values.backstage.extraAppConfig>: nil pointer evaluating interface {}.backstage_

installDir needs to be [assigned to a variable](https://helm.sh/docs/chart_template_guide/variables/#helm) outside of the range block